SQF/dayz_code/actions/AdvancedTrading/functions/z_at_changeBuySell.sqf

```diff
@@ -19,11 +19,11 @@ if(Z_Selling)then{
 	if (Z_CategoryView) then {
 		(findDisplay Z_AT_DIALOGWINDOW displayCtrl Z_AT_FILTERBUTTON) ctrlSetText localize "STR_EPOCH_VIEW";
 		(findDisplay Z_AT_DIALOGWINDOW displayCtrl Z_AT_BUYABLELIST) ctrlSetFontHeight 0.026; // Slightly bigger in category view	
-		(findDisplay Z_AT_DIALOGWINDOW displayCtrl Z_AT_BUYABLELIST) ctrlSetPosition [(0.21* safezoneW + safezoneX),(0.43 * safezoneH + safezoneY),(0.13 * safezoneW),(0.30 * safezoneH)];	
+		(findDisplay Z_AT_DIALOGWINDOW displayCtrl Z_AT_BUYABLELIST) ctrlSetPosition [(0.21* safezoneW + safezoneX),(0.41 * safezoneH + safezoneY),(0.13 * safezoneW),(0.30 * safezoneH)];	
```

Comment:
If you want to do this please remove both ctrlSetPosition lines. Line 22 and line 16. The only purpose of them was to move the categories list down a bit so it wasn't squished against the top. But if you think it looks better like this it doesn't bother me either way.